### PR TITLE
PORTALS-2616: border radius on account settings table of contents

### PR DESCRIPTION
**Pull request by hallieswan**
main | feature
:---:|:---:
main_authenticated_myaccount | feature_authenticated_myaccount

### CODE REVIEW

**Review by nickgros — APPROVED** [review]
Fine with this simple change but something seems off

File: apps/SageAccountWeb/src/style/_AccountSetting.scss [hunk 1]
```diff
@@ -16,11 +16,16 @@
...
     top: 60px;
     align-self: flex-start;
+    li:first-child {
+      border-radius: 3px 3px 0px 0px;
+    }
+    li:last-child {
+      border-radius: 0px 0px 3px 3px;
+    }
     li {
       background-color: #fff;
```
[the remark this answers is not in the document]
Comment (hallieswan) [reply apps/SageAccountWeb/src/style/_AccountSetting.scss]: It looks like the border-radius and box-shadow are different for the table of contents than in the theme, but we can get the background color, font size, and hover effect automatically -- see next commit and updated screenshots in the issue description

File: apps/SageAccountWeb/src/style/_AccountSetting.scss [hunk 2]
```diff
@@ -6,29 +6,25 @@
...
-    border-radius: 3px;
     position: -webkit-sticky; /* for Safari */
     position: sticky;
     top: 60px;
     align-self: flex-start;
-    li {
-      background-color: #fff;
-      font-size: 14px;
+    padding-top: 10px;
+    padding-bottom: 20px;
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/style/_AccountSetting.scss:22]: Match padding in [Figma](https://www.figma.com/file/TOcmJ2QHr51v8Huv3wIec9/Sage-Registration-App?node-id=1755-45706&t=exIPmwrsFpKOL1pJ-4) more closely

File: apps/SageAccountWeb/src/style/_AccountSetting.scss [hunk 3]
```diff
@@ -6,29 +6,25 @@
...
-    min-width: 200px;
     margin-right: 60px;
+    white-space: nowrap;
     box-shadow: 0px 8px 4px rgba(0, 0, 0, 0.025),
       0px 4px 2px rgba(0, 0, 0, 0.025), 0px 2px 1px rgba(0, 0, 0, 0.025);
-    border-radius: 3px;
     position: -webkit-sticky; /* for Safari */
     position: sticky;
     top: 60px;
     align-self: flex-start;
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/style/_AccountSetting.scss:20]: To make TOC sticky

**Review by nickgros — APPROVED** [review]

File: apps/SageAccountWeb/src/components/StyledComponents.ts [hunk 4]
```diff
@@ -31,8 +31,7 @@ export const StyledInnerContainer: StyledComponent<PaperProps> = styled(Paper, {
   width: '900px',
   margin: '0 auto',
   display: 'flex',
-  boxShadow: '0px 2px 6px rgba(53, 58, 63, 0.1)',
-  borderRadius: '5px',
+  overflow: 'hidden',
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/components/StyledComponents.ts:34]: Fix border-radius on StyledInnerContainer and use Paper styling from theme overrides

main | feature
:---:|:---:
main_styledcontainer | feature_styledcontainer

File: apps/SageAccountWeb/src/style/_AccountSetting.scss [hunk 5]
```diff
@@ -6,36 +6,30 @@
...
-      &:hover {
-        font-weight: bold;
-      }
     }
   }
   &.main-panel {
     background-color: #fff;
     padding: 54px;
-    box-shadow: 0px 2px 6px rgba(53, 58, 63, 0.1);
-    border-radius: 5px;
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/style/_AccountSetting.scss:38]: Use overridden Paper theme to get box-shadow and border-radius

File: apps/SageAccountWeb/src/style/_Register.scss [hunk 6]
```diff
@@ -24,8 +24,8 @@
   }
   .thankYouContainer {
     background: #fefefe;
-    box-shadow: '0px 2px 6px rgba(53, 58, 63, 0.1)';
-    border-radius: '5px';
+    box-shadow: 0px 2px 6px rgba(53, 58, 63, 0.1);
+    border-radius: 5px;
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/style/_Register.scss:28]: Fix box-shadow and border-radius:

main | feature
:---:|:---:
main_register1 | feature_register1

File: apps/SageAccountWeb/src/components/SageResourcesPage.tsx [hunk 7]
```diff
@@ -22,9 +22,6 @@ export const SageResourcesPage = (props: SageResourcesPageProps) => {
         sx={{
           margin: '0 auto',
           width: '900px',
-          backgroundColor: '#FFF',
-          boxShadow: '0px 2px 6px rgba(53, 58, 63, 0.1)',
-          borderRadius: '5px',
```
Comment (hallieswan) [inline apps/SageAccountWeb/src/components/SageResourcesPage.tsx:27]: Use overridden Paper theme to get box-shadow and border-radius - no visible changes to styling

main | feature
:---:|:---:
main_sageresources | feature_sageresources

**Review by nickgros — APPROVED** [review]

**Pull request merged**